core-feature-pack/src/main/resources/content/bin/domain.sh

```diff
@@ -96,6 +96,26 @@ if [ "x$JAVA" = "x" ]; then
     fi
 fi
 
+$JAVA --add-modules=java.se -version > /dev/null 2>&1 && MODULAR_JDK=true || MODULAR_JDK=false
```

Comment:
MODULAR_JDK variable is not used in domain.sh, so domain is unable to start with JDK8:
```
[mkopecky@dhcp-10-40-4-197 bin]$ ./domain.sh 
=========================================================================

  JBoss Bootstrap Environment

  JBOSS_HOME: /home/mkopecky/erase8/wildfly

  JAVA: /home/mkopecky/base/jdk1.8.0_162/bin/java

  JAVA_OPTS: -server -Xms64m -Xmx512m -XX:MaxMetaspaceSize=256m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED --add-modules=java.se

=========================================================================

Unrecognized option: --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
[mkopecky@dhcp-10-40-4-197 bin]$ 
```